### ISSUE TITLE
Feature:  Support a flag to delete Sensitive Headers

### ISSUE THREAD

**Issue by RisingPhorce** [jcmoraisjr/haproxy-ingress#319]
Per reverse proxy best practices, please consider adding a flag to delete headers that may often expose internal application/framework versions and potentially vulnerable targets.  Perhaps a list of headers or regexs.

rspidel ^Server:.*$
rspidel ^X-Powered-By:.*$
rspidel ^X-AspNet-Version:.*$

**Comment by jcmoraisjr** [comment]
Hi, how'd you see the interface of such option? The number and type of headers may change a lot from one workload to another, I cannot see a way to implement a generic solution. Perhaps you are just talking about a kind of syntax sugar for `config-backend` where one would name a list of blacklisted headers instead of haproxy keywords?

**Comment by RisingPhorce** [comment]
Yes, I was considering a list of blacklisted headers.  However, I would like support for configuring it on the frontend so that they are globally banned for all backends.
